lookout/style/format/postprocess.py

```diff
@@ -56,6 +56,7 @@ def filter_uast_breaking_preds(
     :param vnode_parents: `VirtualNode`-s' parents mapping as the LCA of the closest
                            left and right babelfish nodes.
     :param node_parents: Parents mapping of the input UASTs.
+    :param rule_winners: Winning rule for each sample.
```

Comment:
numpy array of `Rule`-s (just to make sure)?

Reply:
numpy array of rule indices, I'll update the docstring since we can't give proper numpy.ndarray type hints :(